src/transformers/models/auto/configuration_auto.py

```diff
@@ -528,6 +530,7 @@
         ("mobilevit", "MobileViT"),
         ("mobilevitv2", "MobileViTV2"),
         ("mpnet", "MPNet"),
+        ("mra", "Mra"),
```

Comment:
```suggestion
        ("mra", "MRA"),
```
The name should be spelled with casing here.